pkgs/data/icons/super-tiny-icons/default.nix

```diff
@@ -0,0 +1,46 @@
+{ pkgs, lib, stdenvNoCC, fetchFromGitHub, }:
+stdenvNoCC.mkDerivation rec {
+  pname = "super-tiny-icons";
+  version = "20221107-${lib.strings.substring 0 7 rev}";
+  rev = "b4d5a3be04c99ec0a309ac9e0d0b21207c237c7d";
+
+  src = fetchFromGitHub {
+    owner = "edent";
+    repo = "SuperTinyIcons";
+    inherit rev;
+    sha256 = "670ndAuBrZBr7YOTQm2zHJfpiBC56aPk+ZKMneREEoI=";
+  };
+
+  outputs = [ "out" "doc" ];
+  phases = [ "installPhase" ];
```

Comment:
```suggestion
```
Doing this in packages is forbidden. Please disable other phases if they cause issues.

Reply:
That is good to know :+1: